Sources/Filters.swift

```diff
@@ -13,6 +13,33 @@ enum Filters {
     case invalidOption(option: String)
   }
 
+  /// Parses filter input value for a string value, where accepted objects must conform to
+  /// `CustomStringConvertible`
+  ///
+  /// - Parameters:
+  ///   - value: an input value, may be nil
+  /// - Throws: Filters.Error.invalidInputType
+  static func parseString(from value: Any?) throws -> String {
+    guard let stringArg = value as? LosslessStringConvertible else {
+      throw Error.invalidInputType
+    }
+    return String(describing: stringArg)
+  }
+
+  /// Parses filter arguments for a string value, where accepted objects must conform to 
+  /// `CustomStringConvertible`
+  ///
+  /// - Parameters:
+  ///   - arguments: an array of argument values, may be empty
+  ///   - index: the index in the arguments array
+  /// - Throws: Filters.Error.invalidInputType
+  static func parseStringArgument(from arguments: [Any?], at index: Int = 0) throws -> String {
```

Comment:
To keep the naming consistent with the other filters, we could consider making the index mandatory (no default value). And modify the other parse argument functions (bool/enum) to require an index as well.

Reply:
Honestly I don't see the benefit for this, we'll increase the number of repeatable code in order to have naming consistency. I think it doesn't worth it 😄